project.clj

```diff
@@ -3,7 +3,8 @@
   :url "https://github.com/RickMoynihan/lein-tools-deps"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/tools.deps.alpha "0.3.260"]]
-
+  :dependencies [[org.clojure/clojure "1.9.0"]
+                 [org.clojure/tools.deps.alpha "0.3.260"]]
   ;:plugins [[thomasa/mranderson "0.4.8"]]
-  :eval-in-leiningen true)
+  :eval-in-leiningen true
+  :min-lein-version "2.8.1")
```

Comment:
You might want to change this. If you change, you have to change also the version in travis (see above comments, otherwise we test something different that what we claim to support.